sdcm/logcollector.py

```diff
@@ -340,10 +340,10 @@ def install_phantom_js(self):
                 tar xvfj {phantomjs_tar}
             """.format(**locals()))
             localrunner.run("bash -ce '%s'" % install_phantom_js_script)
-            localrunner.run(
-                "cd {0.phantomjs_base} && sed -e 's/200);/10000);/' examples/rasterize.js |grep -v 'use strict' > r.js".format(self))
         else:
             LOGGER.debug("PhantomJS is already installed!")
+        localrunner.run(
+            "cd {0.phantomjs_base} && sed -e 's/200);/10000);/' examples/rasterize.js |grep -v 'use strict' > r.js".format(self))
```

Comment:
I removed the 'use strict' for workaround an error in generating the snapshot.
The issue had been fixed by https://github.com/ariya/phantomjs/pull/14824.

However the sed for increasing timeout is still needed, it's very slow to load all metrics.

Reply:
But there is no official release of phantomjs after 2016-01-24, maybe we can save fixed examples/rasterize.js to data_dir/ like `data_dir/share_snapshot.js`?

Reply:
Sounds reasonable, until we can replace phantomjs if it's not clearly maintained